Builders/FormTreeBuilder.php

```diff
@@ -84,17 +84,17 @@ private function createNodeFromView(FormView $view)
      */
     private function hasChildrenWithLabel(FormView $view)
     {
-    	if (!isset($view->vars['compound']) || !$view->vars['compound']) {
+        if (!isset($view->vars['compound']) || !$view->vars['compound']) {
             return false;
-    	}
+        }
 
-    	foreach ($view->vars['block_prefixes'] as $prefix) {
-            if (in_array($prefix, $this->noChildren)) {
+        foreach ($view->vars['block_prefixes'] as $prefix) {
+            if (\in_array($prefix, $this->noChildren, true)) {
```

Comment:
Making it strict now is likely to cause obscure troubles, we should either keep it is as is or trigger a deprecation notice when a non strict match occur

Reply:
(`in_array` keeps optimized without the third argument)

Reply:
You're right, I'll remove the third parameter